### DISCUSSION TITLE
A few suggestions

### DISCUSSION THREAD

**Discussion by rogervc** [rbreu/beeref#13] [Ideas & General Discussion]
Hello,
First of all, thank you for making BeeRef, its great to finally have an open source reference tool.
I have been trying the app for the last few days to see if it was a viable replacement to PureRef, (it is), and i thought of a few suggestions that could be useful:

- Some images, especially ones with low resolution, appear jagged. Would it be possible to implement some sort of anti-aliasing to improve that?

- Loading save files by dropping them on the program.

- A greyscale filter for the images.

- A simple settings window to assign shortcuts, change current ones and maybe change the background color.

- I noticed that, when you create a new scene or exit the program, there's no warning dialog for unsaved scenes.

- There's a bit of a "bug" when hiding the title bar. The app leaves a space on the bottom of the same height as the title bar. This happens in Kubuntu, i don't know if it happens in Windows or Mac-os. Its easy to fix maximizing the window again, so its not really a big deal.

-  Also, it would be great if you could make a Snap or Flatpak of the app.

I will post most suggestions in the future when i have them.

In its current stat BeeRef is a good tool, and i will use it as my main reference app from now on.

**Comment by rbreu** [comment]
Thank you for the feedback!

Configuration options are definitely planned! Anti-alias is something that needs to be configurable, since for pixel art you don' want it. 

About the title bar hiding, that's when when the window is maximized? I can look into it. But you can also just use the full screen option (F11) in that case.
